quarkus-extension/simple-rest-example/README.md

```diff
@@ -0,0 +1,21 @@
+# Camunda Platform Runtime Quarkus Example
+
+This project uses Quarkus, the Supersonic Subatomic Java Framework.
+
+If you want to learn more about Quarkus, please visit its website: https://quarkus.io/ .
+
+## Packaging and running the application
```

Comment:
It would also be useful to add a requirements section. For example:
```suggestion
## Requirements

* Java 11+
* Maven 3.8.1+

## Packaging and running the application
```

Reply:
👍